### ISSUE TITLE
Code the Resources page

### ISSUE THREAD

**Issue by HeatherRemacle** [bcgov/digital.gov.bc.ca#82]
## Acceptance Criteria
- Implement Designs (using established styles associated with the home page)
- Make sure page loads properly and quickly
- 

#### Acceptance Criteria

- [x] Page contains all content as shown on [Wireframes](https://miro.com/app/board/o9J_kslEIZ4=/?moveToWidget=3074457348200785828&cot=13) 
- [x] The lInk for Video Communication Platforms must remain the same as it is on the current site (https://digital.gov.bc.ca/communication-platforms/)
- [x] Content renders in one-column layout on mobile phone displays, 12-column layout on larger/desktop

CSS for the following matches specs defined in the Design System:

- [ ] [Colour Palette](https://developer.gov.bc.ca/Design-System/Colour-Palette)
- [ ] [Typography](https://developer.gov.bc.ca/Design-System/Typography)
- [ ] [Links](https://developer.gov.bc.ca/Design-System/Links)
- [ ] [Icons](https://developer.gov.bc.ca/Design-System/Icons)
- [x] [Header - Basic](https://developer.gov.bc.ca/Design-System/Header-Basic)
- [x] [Footer](https://developer.gov.bc.ca/Design-System/Footer-Basic)
- [x] [Navigation Bar - Basic](https://developer.gov.bc.ca/Design-System/Navigation-Bar-Basic) (collapses to hamburger)

#### D.O.D.

- [x] Page can be viewed online in a DEV, TEST or PROD environment
- [x] Page loads quickly

**Comment by Dianadec** [comment]
Navigation should be completed prior to this one.

**Comment by ryleyw** [comment]
Going to add the functionality to switch between home and resources to this ticket, just so we don't have a site for every page.

**Comment by HeatherRemacle** [comment]
I have several comments to address in Figma to reorient some of the content given some noticeable duplication when I'm clicking through (yay prototypes!). This is also leading to some relatively minor requested changes on the home page.

Also, I'm looking for all the acceptance criteria and DoD to be met above - in particular the links.

**Moving this back to in Progres**s @ryleyw  @thegentlemanphysicist @parulmishra @singhinderdeep - let's pitch in to get this closer to done.

@Dianadec - Perhaps we should be creating a new task to do the edits on the home page? It would be very small points (which is fine... we have very few of those).

@mark-a-wilson - I don't want to create more work in going back to docs to ask for edits. I think once we go live, those can likely be retired - it is easier for me to use figma, or screenshots or Miro. The content templates are helpful for starting from scratch from my perspective - but let me know if you have other reasons for keeping those up to date.

**Comment by mark-a-wilson** [comment]
@HeatherRemacle - seeing your comments in Figma works fine. To avoid rework on Ryley's part, I'd suggest we modify our process slightly: 

- Use the Google Docs to draft content and keep track of target audience and what we're trying to accomplish with that page, but NOT to seek your final approval on content 
- Once content is drafted, handoff to Inderdeep to incorporate all content in the wireframe
- You can review and comment on the design+content in Figma; we can make adjustments
- Once you have signed off on the wireframe, then we can pass to Ryley for development, confident that no more content edits will be requested

cc @ryleyw @singhinderdeep @Dianadec

**Comment by HeatherRemacle** [comment]
I'm happy to try this out if the team prefers that process. I see value in being able to edit directly (as in pitching in to the text editing process rather than asking for someone to repeat my comments). Does it make sense for me to have edit access in Figma? Is that possible?

**Comment by mark-a-wilson** [comment]
I think that would be fine - @singhinderdeep, what do you think?

**Comment by singhinderdeep** [comment]
Should be fine. Have given edit access to @HeatherRemacle

**Comment by Dianadec** [comment]
Another card will be created to adjust the design to design system specifications. AC is still being clarified as part of the team's process. This card will be closed (as per our Sprint Planning discussion July 2)
